dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/RequestParamsValidator.java

```diff
@@ -283,7 +284,7 @@ public static void validateOrderParams(List<OrderCriteria> order, Set<String> su
   /**
    * Validate the {@code filter} request parameter in change log tracker exporters. Allowed filter
    * values are {@code supportedFieldNames}. Only one field name at a time can be specified. If the
-   * endpoint supports UIDs use {@link #parseFilters(String)}.
+   * endpoint supports UIDs use {@link FilterParser#parseFilters(String)}.
```

Comment:
`parseFilters(String)` method in this class can now be removed, right?

Reply:
of course 😂 🤦🏻